pygitguardian/client.py

```diff
@@ -264,15 +264,15 @@ def get(
     def post(
         self,
         endpoint: str,
-        data: Union[Dict[str, Any], List[Dict[str, Any]], None] = None,
+        json: Union[Dict[str, Any], List[Dict[str, Any]], None] = None,
```

Comment:
It seems that the tests use cassettes and don't match on request's body. Can you ensure that you don't break anything here by removing the cassettes and verify that tests behave correctly ?